docs/src/other/glossary.md

```diff
@@ -36,7 +36,10 @@ You can also configure them manually.
 ## Inactive environment
 
 An environment that isn't deployed.
-You can activate an inactive environment from the environment configuration page on the Platform.sh management console.
+To activate an inactive environment:
+
+* In the console, go to the **Settings** page for that environment.
```

Comment:
```suggestion
* In the [management console](https://console.platform.sh) of your project, go to the **Settings** page for that environment.
```

Reply:
I don't think "console of your project" really works, because they aren't multiple consoles for different projects. But I can add a step for more clear navigation.